src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -133,7 +133,7 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
             SubsystemVersion subsystemVersion = default(SubsystemVersion),
             string runtimeMetadataVersion = null,
             bool tolerateErrors = false,
-            bool includePrivateMembers = false,
+            bool includePrivateMembers = true,
```

Comment:
Note: I think this default value makes more sense, but it could be considered a breaking change. Since the parameter did nothing (before this refout work), I think that is ok.